boards/atmega256rfr2-xpro/Makefile.include

```diff
@@ -2,15 +2,10 @@
 PORT_LINUX ?= /dev/ttyACM0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 BAUD ?= 115200
-include $(RIOTMAKE)/tools/serial.inc.mk
-
 # Use EDBG (xplainedpro) programmer with avrdude
 PROGRAMMER ?= xplainedpro
 
-# This board can be reset via avrdude
-RESET ?= avrdude -c $(PROGRAMMER) -p m256rfr2
 # Use edbg interface for debugging
 DEBUGSERVER_INTERFACE ?= --edbg
 
-include $(RIOTMAKE)/tools/avrdude.inc.mk
```

Comment:
This board does not include `common/arduino-atmega/Makefile.include`, so I think this is needed here.

Reply:
nvm, I didn't realize you expanded `common/atmega/Makefile.include`.